source/core/parameters_lagrangian.cc

```diff
@@ -861,9 +949,9 @@ namespace Parameters
     BoundaryMotion<dim>::parse_boundary_motion(ParameterHandler &prm)
     {
       const unsigned int boundary_id = prm.get_integer("boundary id");
-      const std::string  motion_type = prm.get("type");
```

Comment:
Why change this part? Please keep the dynamic cylinder motion as a seperate thing for the time being.

Reply:
Do not remember erasing this line ...

Reply:
Ok